desktop/apps/consignments2/templates/submission_flow.jade

```diff
@@ -0,0 +1,12 @@
+extends ../../../components/main_layout/templates/minimal_header
```

Comment:
🤔  one day we might want to consider refactoring these layout components into React (maybe with jade wrappers for legacy purpose) and get us a step closer to being able to extract to for use in external Reaction-Force-like or otherwise apps.